docs/plugin-proposal-pipeline-operator.md

```diff
@@ -52,7 +52,7 @@ Object.keys(envars)
   |> console.log(^^);
 ```
 
-From [jquery/src/core/init.js][].
+From [jquery/src/core/init.js](https://github.com/jquery/jquery/blob/acb7c49c8d42f601fa347661b1118959079f6b52/src/core/init.js#L51-L55).
```

Comment:
Currently the second link is not working: https://babel.dev/docs/babel-plugin-proposal-pipeline-operator#examples. This might be an MDX caveat.